prow/plank/reconciler.go

```diff
@@ -868,6 +889,10 @@ func pendingTriggeredIndexKeyByName(jobName string) string {
 	return fmt.Sprintf("pending-triggered-named-%s", jobName)
 }
 
+func pendingTriggeredIndexKeyByJobQueueName(jobQueueName string) string {
+	return fmt.Sprintf("pending-triggered-with-job-queue-name-%s", jobQueueName)
```

Comment:
Maybe return an empty string here if `jobQeueName` is an empty string to avoid indexing jobs by queue that do not have a queue set?